en/docs/guides/mfa/x509.md

```diff
@@ -0,0 +1,400 @@
+# Configure X509 Certificate Authenticator
+
+This page guides you through configuring the X509 certificate authenticator and the WSO2 Identity Server using a sample app to demonstrate authentication.
+
+---
+
+## Work with certificates
+
+X509 authentication requires the client to possess a Public Key Certificate (PKC). To know more about ublic Key Certificate (PKC) and Certificate Authority (CA), refer [Working with certificates](TO DO: Concepts)
+
+
+To create a sample certificate and create your own Certificate Authority to sign the certificates, follow the following steps:
+
+1.  The first step is to create the private RSA key:
+
+    ```
+    openssl genrsa -out rootCA.key 2048
+    ```
+
+    Here, the specified key size is 2048 bit. You can specify the key size for your private key.
+
+2.  Based on this key you can now generate an actual certificate which is valid for 10 years using the following command:
```

Comment:
Based on this key, generate an actual...

command: --> command.

Reply:
Avoid using 'you' wherever it can be avoided. Please check other places throughout the guide where it is applicable